packages/react-components/src/components/HighContrastAwareIcon.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Icon, mergeStyles } from '@fluentui/react';
+import React from 'react';
+import { useTheme } from '../theming';
+
+/**
+ * Strings of HighContrastAwareIcon that can be overridden.
+ *
+ * @public
```

Comment:
This should be @private, and should not be exported from the `@internal/react-components` package.

Reply:
I changed both interface and component declaration to private since this is a helper component for the Calling sample only